tfx/components/example_gen/import_example_gen/executor.py

```diff
@@ -69,11 +69,13 @@ def GetInputSourceToExamplePTransform(self) -> beam.PTransform:
 
     @beam.ptransform_fn
     @beam.typehints.with_input_types(beam.Pipeline)
-    @beam.typehints.with_output_types(Union[tf.train.Example, bytes])
+    @beam.typehints.with_output_types(Union[tf.train.Example,
+                                            tf.train.SequenceExample,
+                                            bytes])
     def ImportProtoOrExample(pipeline: beam.Pipeline,
                              exec_properties: Dict[Text, Any],
                              split_pattern: Text) -> beam.pvalue.PCollection:
-      """PTransform to import tf.train.Example records or serialized proto."""
+      """PTransform to import a tf.train.Example, tf.train.SequenceExample or serialized proto."""
```

Comment:
it's a pcollection of xxx, so `s` not `a`

and you may need a full doc comment as it is not fit into a single line

Reply:
understand, i will change this to a full doc comment